torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1841,19 +1841,22 @@ at::Tensor XLANativeFunctions::logsumexp(const at::Tensor& self,
       /*keep_reduced_dimensions=*/keepdim));
 }
 
-at::Tensor XLANativeFunctions::logdet(const at::Tensor& self) {
-  XLA_FN_COUNTER("xla::");
-  return bridge::AtenFromXlaTensor(
-      XLATensor::logdet(bridge::GetXlaTensor(self)));
-}
-
 at::Tensor XLANativeFunctions::xlogy(const at::Tensor& self,
                                      const at::Tensor& other) {
   XLA_FN_COUNTER("xla::");
   return bridge::AtenFromXlaTensor(XLATensor::xlogy(
       bridge::GetXlaTensor(self), bridge::GetXlaTensor(other)));
 }
 
+std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::slogdet(
+    const at::Tensor& self) {
+  XLA_FN_COUNTER("xla::");
+  XLATensor self_tensor = bridge::GetXlaTensor(self);
+  auto outputs = XLATensor::slogdet(self_tensor);
+  return std::make_tuple(bridge::AtenFromXlaTensor(std::get<0>(outputs)),
+                         bridge::AtenFromXlaTensor(std::get<1>(outputs)));
+}
+
```

Comment:
nit: maybe should we move this back to the original place so this PR doesn't show `slogdet` as being modified?